apps/api/src/domain/models/bibliographic-reference/entities/BookBibliographicReferenceData.ts

```diff
@@ -11,11 +22,62 @@ export default class BookBibliographicReferenceData
     readonly type = BibliographicReferenceType.book;
 
     @IsStringWithNonzeroLength()
-    title: string;
+    readonly title: string;
+
+    @ArrayNotEmpty()
```

Comment:
Can we rename this to `IsNonEmptyArray` before exporting it from our lib?

Reply:
Renamed `ArrayNotEmpty` to `IsNonEmptyArray`

Reply:
[Here](https://www.pivotaltracker.com/story/show/182280211) is the story.